src/Uno.UI/Microsoft/UI/Xaml/Controls/TabView/TabView.cs

```diff
@@ -1203,7 +1203,7 @@ public DependencyObject ContainerFromIndex(int index)
 			return null;
 		}
 
-		private object ItemFromContainer(DependencyObject container)
```

Comment:
Does this match the Windows public contract?

Reply:
Yes, I left it out accidentally 😅